sandbox/rpc/go.sum

```diff
@@ -0,0 +1,85 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
is this file supposed to be commited? it looks gross

Reply:
I have no clue. But other go repos track it so...